addons/test_mail/tests/test_ir_actions.py

```diff
@@ -167,7 +167,7 @@ def test_action_send_mail_without_mail_thread(self):
             }
             self.action.with_context(action_ctx).run()
 
-        mail = self.assertMailMail(
+        self.assertMailMail(
```

Comment:
By default, the footer is hidden. So, there will not be any "Powered by ..." mention. I hence adapted the test accordingly.

Reply:
Maybe change the test to have the opposite? Check that the footer isn't added unexpectedly.

Reply:
+1 to test that there is no footer in a default mail :)